everviz/plugins/objectives_plot/objectives_plot.py

```diff
@@ -181,10 +181,11 @@ def set_button_enabled_state(filter_realizations):
                 Input(self.radio_id_mode, "value"),
                 Input(self.realization_filter_check_id, "value"),
                 Input(self.realization_filter_input_id, "value"),
+                Input(self.graph_id, "hoverData"),
             ],
         )
         def update_graph(
-            func_list, radio_value, plot_mode, realizations_check, realizations_input
+            func_list, radio_value, plot_mode, realizations_check, realizations_input, hoverData
```

Comment:
`hoverData` should not be camel case maybe hover_data? or click_data if you go with the replacement on line 184.